public/content/documentation/web/component/search.md

```diff
@@ -0,0 +1,182 @@
+
+2. Desktop screenreader
+
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND 
+   - I use the tab key to move focus to a search input
+      - I HEAR its purpose is clear
+      - I HEAR it identifies itself as a search input
+      - I HEAR the form itself is discoverable with screenreader shortcuts as a search input
+   - THEN when I use the tab key to move focus to the search submit button I HEAR the button is focused
+   - THEN when I use the enter or spacebar key I HEAR the search results are presented
+
+3. Mobile screenreader
+
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+   - I swipe to focus on a search input
+      - I HEAR its purpose is clear
+      - I HEAR it identifies itself as a search input
+      - I HEAR the form itself is discoverable with screenreader
+
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/search](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/search)
+
+
+## Videos
+
+### Android Talkback
+
+<video controls>
+  <source src="media/video/web/search/search-android.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+### iOS VoiceOver
+
+<video controls>
+  <source src="media/video/web/search/search-ios.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+### Windows Jaws Chrome
+<video controls>
+  <source src="media/video/web/search/search-JAWS.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+### Windows NVDA Chrome
+<video controls>
+  <source src="media/video/web/search/search-NVDA.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+### MacOS Voiceover Safari
+<video controls>
+  <source src="media/video/web/search/search-desktop-safari.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+## Code examples
+### Use semantic HTML
+
+- This semantic HTML contains all accessibility features by default. 
+- Include a search button
+
+### Search with autocomplete suggestions
+
+- For search autocomplete see [Autocomplete input with listbox](/checklist-web/listbox-autocomplete/)
+
+```html
+<form role="search">
+  <label for="search">
+    Search this website:
+  </label>
+  <input list="components" type="search" id="search">
+  <datalist id="components"> 
+    <option value="Alert"> 
+    <option value="Animation"> 
+    <option value="Button"> 
+    <option value="Checkbox">
+    <option value="Date picker">
+    <option value="Expander accordion">
+    <option value="Footer / contentinfo">
+    <option value="Form">
+    <option value="Header / banner">
+    <option value="Heading: h1, h2, h3">
+  </datalist>
+
+  <button type="submit">
+    <span class="hidden">Search</span>
+  </button>
+</form>
+```
+
+<!-- Something funky is up with the search button here and I can't figure it out. It is submitting on focus when not using VoiceOver.  -->
```

Comment:
let's add a TODO here in the comment and make a jira ticket also for the focus issue on the search input